draft-ietf-quic-http.md

```diff
@@ -716,8 +716,16 @@ HTTP_RST_CONTROL_STREAM (0x11):
 # Considerations for Transitioning from HTTP/2
 
 HTTP/QUIC is strongly informed by HTTP/2, and bears many similarities.  This
-section points out important differences from HTTP/2 and describes how to map
-HTTP/2 extensions into HTTP/QUIC.
+section describes the approach taken to design HTTP/QUIC, points out important
+differences from HTTP/2, and describes how to map HTTP/2 extensions into
+HTTP/QUIC.
+
+HTTP/QUIC begins from the premise that HTTP/2 code reuse is a useful feature,
+but not a hard requirement.  HTTP/QUIC departs from HTTP/2 primarily where
+necessary to accommodate the differences in behavior between QUIC and TCP (lack
+of ordering, support for streams).
```

Comment:
I like the primarily here.  We don't need to make a hard commitment.

I think that you want to explicitly state that it is our intent to avoid gratuitous changes.  Especially those that might make it difficult or impossible to build extensions across both protocols that have the same semantics.

You might also observe that an extension that is valid in HTTP/QUIC is usually valid in HTTP/2, though not the converse.